docs/http/traffic-policy/actions/custom-response.mdx

```diff
@@ -38,9 +41,13 @@ If this action is executed, no further actions in the traffic policy will be exe
 | Parameter     | &nbsp;                    | Description                                                                                                                                        |
 | ------------- | ------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `status_code` | int                       | The status code of the response.                                                                                                                   |
-| `content`     | string                    | The body of the response.                                                                                                                          |
+| `content`     | string                    | The body of the response. Supports [cel expressions](#templating-syntax) for variable interopolation.                                              |
```

Comment:
Hmmmm I'm thinking maybe these should say `Supports interpolating the results of cel expressions` instead of "variable interplation"
what do you think?

Reply:
Yeah, probably better to just constrain it down to `cel` expressions and leave it at that 👍